app/models/ptable.rb

```diff
@@ -33,6 +33,8 @@ class Ptable < Template
 
   alias_attribute :layout, :template
 
+  attr_exportable :os_family
```

Comment:
`:snippet` should be added here as it's separate for Ptables, or perhaps in the base class.

For ProvisioningTemplates snippet replaces the kind - either the exported "kind" field should state "snippet" (like the metadata in current shipped templates) or simply add snippet: true by adding snippet to the base class.